partials/notifications.hbs

```diff
@@ -16,4 +16,14 @@
 <div class="notification notification-checkout">
     <a class="notification-close" href="javascript:;"></a>
     Success! Your account is fully activated, you now have access to all content.
+</div>
+
+<div class="notification notification-billing-update">
+    <a class="notification-close" href="javascript:;"></a>
+    Success! Your billing data has been updated.
```

Comment:
@peterzimon I pulled copy for this and the one below out of my head so needs a proper review.